src/libraries/CurrencyDelta.sol

```diff
@@ -0,0 +1,40 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.20;
+
+import {Currency} from "../types/Currency.sol";
+
+library CurrencyDelta {
+    uint256 constant CURRENCY_DELTA_SLOT = uint256(keccak256("CurrencyDelta")) - 1;
+
+    function setCurrencyDelta(address locker, Currency currency, int256 delta) internal {
+        uint256 slot = CURRENCY_DELTA_SLOT;
+
```

Comment:
actually could you potentially write a helper called hashKey = _computeKey(locker, currency)? this whole chunk is repeated in both set/get